person.rb

```diff
@@ -0,0 +1,20 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: 'Unknown')
+    @id = rand(9999)
+    @parent_permission = parent_permission
+    @age = age
+    @name = name
+  end
+  attr_reader :id
+  attr_accessor :age, :name
+
+  def can_use_services?
+    (of_age? || @parent_permission)
+  end
+
+  private
+
+  def of_age?
+    true unless @age >= 18
```

Comment:
- [ ] You can remove the unless condition here because it might return undefined if it is false in this case, however we want this method to return either true or false, which it will do if you write it like below:
```
def of_age?
    @age >= 18
  end
```